uiowa/uiowa
# Max input vars menu path override

Resolves #1996 

# How to test
## Reproduce current issue
- Add a `.ddev/php/max_input_vars.ini` file (or you can use an existing `memory_limit.ini` file if you have one). Add (or replace with) the following content:
```
[PHP]
max_input_vars = 3000;
```
- Restart DDEV to have new config take effect.
```
ddev restart && git checkout main && git pull && ddev blt ds --site engineering.uiowa.edu && ddev drush @engineering.local uli admin/structure/menu/manage/main
```
- Make a change moving a menu link to a different parent and save the form. Observe the following error:
> Warning: Unknown: Input variables exceeded 3000.

## Test fix
- Comment out the `max_input_vars = 3000;` line by adding a `#` at the beginning.
```
ddev restart && git checkout max_input_vars_menu && git pull && ddev drush @engineering.local cr
```
- Visit https://engineering.uiowa.ddev.site/admin/structure/menu/manage/main
- Make a change moving a menu link to a different parent and save the form. Observe that there is no error this time.
